test/unit_test/utils.py

```diff
@@ -89,15 +89,6 @@ def get_optimum_model_by_model_path():
     )
```

Comment:
are there any existing UT to cover the failure like [https://github.com/microsoft/Olive/issues/847](https://github.com/microsoft/Olive/issues/847)? 

Basically, the deprecation PR should cover this kind of error before merging. 

So, I suggest add UT for the failure case.

Reply:
The error in the issue was because the deprecation PR incorrectly used the components to make this Pass work. I said the same in a separate PR before it was split into the deprecation PR https://github.com/microsoft/Olive/pull/828#discussion_r1430593966 

Now, this error is not relevant to the optimum pass. It is expected behavior for components to require the missing params. I don't know what unit test we can make here.

Reply:
I mean the CI for deprecation PR should fail to block its merge in this case.

Reply:
Added ut for components use case for optimum conversion pass. The validation check for pytorch model is unrelated to this PR.